testcases/TC01.1-2-fail-1.md

```diff
@@ -0,0 +1,53 @@
+---
+# The layout must be 'testcase'; DO NOT Change
+layout: testcase
+
+# Brief, descriptive title for the test case
+title: Keyboard access - individual keystrokes require specific timing
+
+# The Test Case ID should follow the pattern: 
+# TC[Baseline Test Procedure #]-[Test Instruction #]-
+# [Expected Result (pass/fail/dna)]-[example #], e.g., TC05.1-1-fail-1
+tcid: TC1.1-2-fail-1
+
+# Description of the Test Case, the included code sample, test considerations,
+# and rationale for the expected result according to the applicable ICT
+# Baseline test
+descr: Determine the functionality of visible and hidden interactive interface components (links, form fields, drop down menus, show/hide content, tree views, pop ups/light boxes, iframes, etc.) available using a mouse (hover and/or click). Check that individual keystrokes do not require specific timings for activation.
+
+The code sample includes a requirement to press and hold a key. A successful test should identify a failure against Baseline 1. Keyboard Accessible.
+
+# Reference and link to the applicable ICT Baseline test
+app-baseline: 
+    [1.1 Test Procedure for Keyboard Access](https://section508coordinators.github.io/ICTTestingBaseline/01Keyboard.html#11-test-procedure-for-keyboard-access)
+    
+    **Baseline Test ID:** 1.1-KeyboardAccess
+    
+    **Test Instruction:** 2
+
+# Expected result that the ICT Baseline would predict
+# [Pass | Fail | DNA]
+result: Fail
+
+# Brief description of the rationale for the expected result
+result-descr: > Users must press and hold a key for three seconds to use a function.
```

Comment:
I had YAML errors when building this page. 

- removing the `>` character fixed one of the errors

@bletalien - did you start with a blank template or use an existing testcase file?

Reply:
fixed